tbx/project_styleguide/templates/patterns/navigation/components/includes/subnav-menu-mini.html

```diff
@@ -5,14 +5,12 @@
     <ul class="sub-nav-desktop-mini__list">
         {% for link in links %}
             <li class="sub-nav-desktop-mini__item">
-                {% with children=link.get_children.live.public.in_menu.specific %}
-                    <a
-                        class="sub-nav-desktop-mini__link"
-                        href="{{ link.url }}"
-                    >
-                        {{ link.nav_text }}
-                    </a>
-                {% endwith %}
+                <a
+                    class="sub-nav-desktop-mini__link"
+                    href="{% pageurl link %}"
```

Comment:
This is creating an error in the pattern library: ```ValueError at /pattern-library/render-pattern/patterns/organisms/header/header.html
pageurl tag expected a Page object, got {'nav_text': 'Charities', 'get_children': {'live': {'public': {'in_menu': {'specific': [{'nav_text': 'Charity sub 1'}, {'nav_text': 'Charity sub 2'}, {'nav_text': 'Charity sub 2'}]}}}}}```

Reply:
(I suspect there is quite a bit of updating to do to the navigation yaml files after these changes)

Reply:
all nav should be now working. I've compared it with staging